### DISCUSSION TITLE
Welcome to gridviz Discussions!

### DISCUSSION THREAD

**Discussion by joewdavies** [eurostat/gridviz#89] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wahlatlas** [comment]
It is no exaggeration that gridviz (GridViz?) provided a [eureka moment](https://en.wikipedia.org/wiki/Eureka_effect) for me similar to what D3js did some ten years ago.

I have been working for the German Census team since 2006. In 2017 I talked about [Understanding Grid-Based Census Results](https://ec.europa.eu/eurostat/cros/ntts2017programme/data/abstracts/abstract_91.html) at the NTTS conference.

So far my published visualisation work is mainly based on D3js, see e.g. the map [Energy consumption in the German industry in 2021, by energy carrier](https://service.destatis.de/DE/karten/EnergieverbrauchIndustrie2021.html#!en) or the [calendar visualisation on traffic accidents](https://service.destatis.de/DE/verkehrsunfallkalender/). On my personal website I have been [mapping election results](https://wahlatlas.net), hence my screen-name (Election = Wahl).

As much as I have been [advocating the use of SVG in client side mapping](https://web.archive.org/web/20090926233927/http://www.svgopen.org/2009/registration.php?section=abstracts_and_proceedings#paper_6) ([2009 slides](http://vis.uell.net/svgopen/09/SVG_Strengths_in_Web_Mapping_Neutze2009.pdf)), it has always been clear, that SVG/DOM-based visualisation has limits in terms of cartographic resolution. However, so far I couldn't bring myself to programming in WebGL and Canvas, which is why GridViz comes at the right time, especially given the 2021/22 census round and its results.

Hopefully I can share some GridViz-based maps publicly soon.

Tiling data had been a little headache at first, but helpful and supporting comments in #81 finally lead to a Python/Pandas solution for my preferred workflow, see #95. Also see all the closed issues as an example that novices are treated very welcoming here. 

Apart from #94 I would like to see a way to control pie-chart transparency by ZoomFactor but probably the game-changer for german data was your added support for WMS backgrounds (#84).

**Reply by jgaffuri** [reply]
Thank you, and not only for your message but also to support the development of better web maps ! And to test gridviz ! I look foward seing it used for the amazing 100m resolution data of Destatis.

**Comment by wahlatlas** [comment]
ResizeObserver is a welcome addition in v2.0.33, thanks!

**Comment by jgaffuri** [comment]
Gridtiler v1 is now out:
https://github.com/eurostat/gridtiler

I tested it quite a lot. But more tests are always welcome !

**Comment by wahlatlas** [comment]
![4b51b00431c18849](https://github.com/eurostat/gridviz/assets/4449821/81eca12a-4c03-47f1-ae0d-8251e47757ae)

Here's an example with 100m grid data from the German Census 2011 that shows heating types as pie charts for each grid cell. There is a WMS map as background layer and in between a WebGL layer allows to highlight densely populated areas. Right now you can adjust the population density threshold with the arrow keys of your keyboard. Depending on zoom level each layer changes color and opacity to best facilitate overview versus detailed orientation on the street level.

https://service.destatis.de/shapes/Zensus2011/Heizungsarten_Ewz_100m_Gitter.html#!4318367,3383722,5.48

Navigating the map is supported by a mini map of Germany in the lower left and Zoom/Pan states are rewritten in the URL

**Reply by jgaffuri** [reply]
Great development ! Thanks a lot for sharing. It is amazing how such detailled data at grid level inform on the geography of heating system types. Depending on the region, ruban/rural type, etc.
I think it would help showing aggregated data depending on the zoom level, at 200, 500, 1000m resolution. The app reaches its limit when showing 100m resolution at country level. But it still works !
